_data/families.csv

```diff
@@ -1,8 +0,0 @@
-position,name,display,parent,urlStub
-1,content-areas,Content Areas,Content Areas,content-areas
-2,inputs-controls,Inputs & Controls,Inputs and Controls,inputs-and-controls
-3,nav,Navigation,Navigation,navigation
-4,notify,Notifications,Notifications,notifications
-5,processing,Processing,Processing,processing
-6,search-filters,Search & Filters,Search and Filters,search-and-filters
-7,tables,Tables,Tables,tables
```

Comment:
What is this file used for? Is there code somewhere that might break if this is missing?